sort/challenges-12.test.js

```diff
@@ -0,0 +1,318 @@
+'use strict';
+
+// ------------------------------------------------------------------------------------------------
+// CHALLENGE 1
+//
+// Write a function named sortBackwards that takes in an array of numbers and returns the same array,
+// with the numbers sorted, highest to smallest.
+// ------------------------------------------------------------------------------------------------
+
+const sortBackwards = (nums) => {
+  nums.sort(function(a,b) {
+    return b - a;
+  });
+  return nums;
+};

```

Comment:
you can combine these into one line `return nums.sort( (a,b) => b - a );`

Reply:
updated it!